### PR TITLE
898 stats is undefined on bare metal homepage

### LINKED ISSUE #898: 🐛 'stats' is undefined on bare-metal homepage
### Is there an existing issue for this?

- [X] I have searched the existing issues

### I have checked the wiki

- [X] I have searched the wiki for this problem

### Does this issue exist in the latest version?

- [X] I'm using the latest release

### Describe the bug?

'stats' is undefined error happens when running on bare-metal with no docker. Error doesn't show up when running inside a container.

### To Reproduce

1. Install latest arm without using container
2. visit home page
3. error…

### What OS are you seeing the problem on?

Debian 11

### ARM Release Version or Git hash (You can find this in the arm settings page)

v2.6.53

### Expected behavior?

Show home page

### Relevant log output

```shell
[2023-07-12 22:17:17,241] DEBUG ARM: notifications.arm_nav_notify 3
Traceback (most recent call last):
  File "/home/arm/.local/lib/python3.9/site-packages/flask/app.py", line 2091, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/arm/.local/lib/python3.9/site-packages/flask/app.py", line 2076, in wsgi_app
    response = self.handle_exception(e)
  File "/home/arm/.local/lib/python3.9/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/arm/.local/lib/python3.9/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/arm/.local/lib/python3.9/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/arm/.local/lib/python3.9/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/arm/.local/lib/python3.9/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/arm/.local/lib/python3.9/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/arm/.local/lib/python3.9/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/opt/arm/arm/ui/routes.py", line 78, in home
    return render_template("index.html", jobs=jobs, armname=armname,
  File "/home/arm/.local/lib/python3.9/site-packages/flask/templating.py", line 147, in render_template
    return _render(
  File "/home/arm/.local/lib/python3.9/site-packages/flask/templating.py", line 128, in _render
    rv = template.render(context)
  File "/home/arm/.local/lib/python3.9/site-packages/jinja2/environment.py", line 1291, in render
    self.environment.handle_exception()
  File "/home/arm/.local/lib/python3.9/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/opt/arm/arm/ui/templates/index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/opt/arm/arm/ui/templates/base.html", line 85, in top-level template code
    {% block content %}{% endblock %}</div>
  File "/opt/arm/arm/ui/templates/index.html", line 63, in block 'content'
    {% include 'settings/sysinfo.html' %}
  File "/opt/arm/arm/ui/settings/templates/settings/sysinfo.html", line 1, in top-level template code
    {% block sysinfo %}
  File "/opt/arm/arm/ui/settings/templates/settings/sysinfo.html", line 129, in block 'sysinfo'
    Intel QuickSync: {% if stats['hw_support']['intel'] %}
  File "/home/arm/.local/lib/python3.9/site-packages/jinja2/environment.py", line 455, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: 'stats' is undefined
```

### Anything else?

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by 1337-server**
# Description
Includes HW info (stats) on home page to stop the homepage crashing arm on bare-metal
Fixes github actions check to test UI

Fixes #898 

## Type of change
Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.
Please also list any relevant details for your test configuration.

- [ ] Docker
- [x] Other (Please state here)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have tested that my fix is effective or that my feature works

# Changelog:

Include the details of changes made here
- Include the stats variable into the homepage as it's required to show HW transcode status
- Fix line length for both errors on history and database pages. (pylint build error)

# Logs

N/A UI changes only

Will update and push to ready when I move to docker and retest everything is still working as expected.

### CODE REVIEW

**Comment by 1337-server** [comment]
Ready for review

File: arm/ui/database/database.py [hunk 1]
```diff
@@ -46,7 +46,8 @@ def view_database():
     # Check for database file
     if os.path.isfile(cfg.arm_config['DBFILE']):
         jobs = models.Job.query.order_by(db.desc(models.Job.job_id)).paginate(page=page,
-                                                                              max_per_page=int(armui_cfg.database_limit),
+                                                                              max_per_page=int(
```
Comment (shitwolfymakes) [inline arm/ui/database/database.py:49]: This change has no effect?

File: arm/ui/history/history.py [hunk 2]
```diff
@@ -35,7 +35,8 @@ def history():
         # after roughly 175 entries firefox readermode will break
         # jobs = Job.query.filter_by().limit(175).all()
         jobs = models.Job.query.order_by(db.desc(models.Job.job_id)).paginate(page=page,
-                                                                              max_per_page=int(armui_cfg.database_limit),
+                                                                              max_per_page=int(
```
Comment (shitwolfymakes) [inline arm/ui/history/history.py:38]: This change has no effect?

**Review by shitwolfymakes — CHANGES_REQUESTED** [review]

File: arm/ui/database/database.py (hunk 1 above)
Comment (1337-server) [reply arm/ui/database/database.py:49]: Line length for the Quality checks. Without this change, it causes the checks to fail.

Max line length is 120, but without this change, the total line length is 121.

File: arm/ui/history/history.py (hunk 2 above)
Comment (1337-server) [reply arm/ui/history/history.py:38]: Same as above

File: arm/ui/database/database.py (hunk 1 above)
Comment (shitwolfymakes) [reply arm/ui/database/database.py:49]: Ah, fair enough

**Review by shitwolfymakes — APPROVED** [review]

**Pull request merged**